resources/views/layouts/app.blade.php

```diff
@@ -32,7 +32,8 @@
         <nav class="navbar navbar-expand-md navbar-light shadow-sm" style="background-color: #E7DA3D;">
             <div class="container">
                 <a class="navbar-brand" href="{{ url('/') }}">
-                    <img class="logo" src="../storage/Logo_ExploreEase_temporaly.png" alt="Logo">
+                    <img class="logo" src="../storage/Logo.temporaly.png" alt="Logo" style="width:32px; height:32px;">
```

Comment:
please include the image in your pull request because this implementation will only work on your end since you have the image in your local but won't work on our end. My suggestion for this is to create a directory inside `public` and name it `assets` and add the image file for the logo inside. and change the path in your `img` tag

Reply:
and also please resolve the conflicts indicated below.

Reply:
@kredo-school
Thank you for your comment. In regards to assets, do you mean this folder?
[assets folder.pdf](https://github.com/kredo-school/14th_explore_ease/files/13266386/assets.folder.pdf)

Reply:
@sumosumo727 that folder is for the build files for js and css
there should be a different folder relative to the `build` directory
should be `/public/assets/` then maybe also add another folder inside 
name the folder `images` and same the logo inside